Buildings/Controls/OBC/ASHRAE/G36/AHUs/SingleZone/SetPoints/ReliefDamper.mo

```diff
@@ -1,25 +1,20 @@
-﻿within Buildings.Controls.OBC.ASHRAE.G36.AHUs.SingleZone.SetPoints;
+within Buildings.Controls.OBC.ASHRAE.G36.AHUs.SingleZone.SetPoints;
 block ReliefDamper "Control of actuated relief  dampers without fans"
   parameter Real minRelDamPos(min=0, max=1, unit="1") = 0.1
-    "Relief damper position that maintains a building pressure of 0.05 inchWC (12.44 Pa) while the system is at MinPosMin 
-    (i.e., the economizer damper is positioned to provide MinOA while the supply fan is at minimum speed)."
+    "Relief damper position maintaining building static pressure at setpoint while the system is at MinPosMin"
     annotation(Evaluate=true, Dialog(group="Nominal parameters"));
   parameter Real maxRelDamPos(min=0, max=1, unit="1") = 0.9
-    "Relief damper position that maintains a building pressure of 
-    0.05 inchWC (12.44 Pa) while the economizer damper is fully open 
-    and the fan speed is at cooling maximum."
+    "Relief damper position maintaining building static pressure at setpoint while the system is at MaxPosMax"
```

Comment:
```MinPosMin``` starts with lower case m in line 12. This could also cause issues if we change the variable name at some point, so maybe it's worth to write it out.